### DISCUSSION TITLE
IXLWorksheet.Range.DataType Missing from versions 0.100.0 and Up

### DISCUSSION THREAD

**Discussion by vsrzzone** [ClosedXML/ClosedXML#2359] [Q&A]
Hi, I dont see a way to set the DataType option in IXLWorksheet.Range ... Is it moved elsewhere or is there a workaround for it?

**Comment by jahav** [comment]
That possibility was removed, because it makes no sense to set data type without a value. I have "Yes" as text, now change it to bool.... Conversion is let to the user (see [Cell value is now strongly typed](https://github.com/ClosedXML/ClosedXML/releases/tag/0.100.0)).

Cell value is now represented by `XLCellValue` with implicit conversions from standard types.

Take value of a cell, do the conversion as desired and set the value back to the cell.
